pkgs/tools/misc/coreboot-configurator/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub, inkscape, meson, ninja, pkg-config, libyamlcpp, nvramtool, qt5 }:
```

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, inkscape
, meson
, ninja
, pkg-config
, libyamlcpp
, nvramtool
, wrapQtAppsHook
, qtbase
, qtsvg
}:
```
Also, is Inkscape really required here?

---

Yes, it's used to generate some icons:

https://github.com/StarLabsLtd/coreboot-configurator/blob/37c93e7e101a20f85be309904177b9404875cfd8/src/resources/meson.build#L22-L43